pkg/controller/test-images-distributor/test_images_distributor_test.go

```diff
@@ -376,6 +382,28 @@ func TestReconcile(t *testing.T) {
 				return nil
 			},
 		},
+		{
```

Comment:
Looks like we treat pullspecs without `/` in a special way, we should test that behavior too.

Reply:
This is actually a bug. My bad.
After copying from https://github.com/openshift/ci-tools/pull/1067/files#diff-195d249eb017f311ef48ddb48944ae74R385, I forgot the change it to true.